FlaskApp/api/user.py

```diff
@@ -0,0 +1,30 @@
+from datetime import datetime 
+class User:
+    #class variable holds all users
+    users = []
```

Comment:
You have a class user for users I guess? but I cant see where your `__init__` is  it makes me wonder how are you going to instantiate the user when you get the values of the user.